test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetListCommandTest.cs

```diff
@@ -101,6 +101,132 @@ public void ListCommand_WithUserSpecifiedSource()
             }
         }
 
+        // Skip: https://github.com/NuGet/Home/issues/11576
+        [PlatformTheory(Platform.Windows, Platform.Linux, SkipMono = true)]
+        [InlineData(false, false)]
+        [InlineData(false, true)]
+        [InlineData(true, false)]
+        [InlineData(true, true)]
+        public void ListCommand_WithLocalSource_WithMatchText_ReturnLatestVersionOnlyPerPackageId(bool includeMatchingText, bool preRelease)
```

Comment:
Are there no equivalent tests? 
Can you make sure you're not duplicating tests.

Reply:
Note that you don't need all combinations either. 

There's a precedence between matching text and prerelease, so you don't really need the false, true and false, false both.

Reply:
Actually, they're not really duplicate tests. 
I removed slow integration tests.